plugin/plugin.py

```diff
@@ -85,9 +85,8 @@ class CopilotPlugin(NpmClientHandler):
         server_directory,
         "node_modules",
         "copilot-node-server",
-        "copilot",
-        "dist",
-        "agent.js",
+        "bin",
+        "copilot-node-server",
```

Comment:
Using a fixed path binary is better. Introduced in https://github.com/jfcherng/copilot-node-server/pull/2/files